lib/define/mysql.js

```diff
@@ -269,6 +279,9 @@ MysqlConnection.prototype.select = function(table, fields, where0, callback) {
     columns: fields,
     where: where0,
   })
+  if (!sql) {
```

Comment:
you copied this section of code many times. put them into _startQuery to avoid duplication